pilot/pkg/config/file/store.go

```diff
@@ -282,12 +282,26 @@ func (s *KubeSource) parseContent(r *collection.Schemas, name, yamlText string)
 		}
 
 		chunk := bytes.TrimSpace(doc)
-		r, err := s.parseChunk(r, name, lineNum, chunk)
+		resource, err := s.parseChunk(r, name, lineNum, chunk)
```

Comment:
I would justchange parseChunk to natively handle List, parsing error feels a bit problematic?

Reply:
@howardjohn Done. Thanks